app/models/user.rb

```diff
@@ -15,14 +15,18 @@ class User < ApplicationRecord
   validates :google_id, presence: true, uniqueness: true
   validates :google_image_url, presence: true, uniqueness: true
   validates :zip_code, presence: true, numericality: true
+
   validates :summary, presence: true
```

Comment:
Let's remove this blank space before merging.

Reply:
^^